src/SelfUpdate/ComposerVersionChecker.php

```
@@ -68,6 +67,15 @@ public function check(): ?VersionCheckResult
         return new VersionCheckResult($currentVersion, $latestVersion);
     }
 
+    /**
+     * Returns the installed DevTools version without running external Composer commands.
+     */
+    public function resolveCurrentVersion(): ?string
+    {
+        return InstalledVersions::getPrettyVersion(self::PACKAGE)
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Composer\InstalledVersions' in method 'resolveCurrentVersion'. (undefined)

(StaticAccess)